ethereum/executionclient/src/main/java/tech/pegasys/teku/ethereum/executionclient/methods/JsonRpcRequestParams.java

```diff
@@ -51,6 +51,11 @@ public <T> Optional<T> getOptionalParameter(final int index, final Class<T> para
     return Optional.of((T) param);
   }
 
+  @SuppressWarnings({"unchecked", "unused"})
```

Comment:
`unused`?

Reply:
Essentially the Class parameter in the method is unused but helps the compiler understand what type is returned, so can pipe orElse() afterwards.